### PR TITLE
Fix telemetry being disabled for protocols used outside of ports tab

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by klutvott123 — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by LoakAudio** [comment]
Hi all, tested with the linux portable version : no more Telemetry vanishing when playing with Ports tab.
Thanks gentlemen.

**Pull request merged**
